src/transformers/training_args.py

```diff
@@ -374,6 +376,12 @@ class TrainingArguments:
             "help": "When resuming training, whether or not to skip the first epochs and batches to get to the same training data."
         },
     )
+    tokenizers_parallelism: bool = field(
```

Comment:
that's a bit out-of-scope for this PR @TevenLeScao - can we maybe only do the required changes to make `run_mlm_flax.py` work? I would prefer not to touch the general `training_args.py` file in this PR. Don't think @sgugger would be too happy about it

Reply:
Yeah, I was a bit conflicted about that one - without a tokenizers_parallelism argument I find that `run_mlm_flax` outputs a ton of tokenizers-related warnings. I had it in `training_args`, but I can also just hard-code it in `run_mlm_flax`.